tools/SetupFlow/DevHome.SetupFlow/ViewModels/AppManagementViewModel.cs

```diff
@@ -30,9 +30,6 @@ public partial class AppManagementViewModel : SetupPageViewModelBase
     [ObservableProperty]
     private ObservableObject _currentView;
 
-    [ObservableProperty]
-    private bool _searchBoxEnabled;

```

Comment:
Search is always enabled as now we will wait for the connection to be established (using semaphores) or attempt to recover on failures